### ISSUE TITLE
META should have closing bracket

### ISSUE THREAD

**Issue by Anton-V-K** [microsoft/XmlNotepad#115]
XSL Output window can't show anything because `META` tag has no closing bracket.
An example of generated HTML:
```
<head>
  <META http-equiv="Content-Type" content="text/html; charset=utf-16">
  <title>Activity Monitor Log</title>
  <style type="text/css">
            body{ text-align: left; width: 100%;  font-family: Verdana, sans-serif; }

            table{ border: none;  border-collapse: separate;  width: 100%; }

            tr.title td{ font-size: 24px;  font-weight: bold; }

            th{ background: #d0d0d0;  font-weight: bold;  font-size: 10pt;  text-align: left; }
            tr{ background: #eeeeee}
            td, th{ font-size: 8pt;  padding: 1px;  border: none; }

            tr.info td{}
            tr.warning td{background-color:yellow;color:black}
            tr.error td{background-color:red;color:black}
            
            span {text-decoration:underline}
            a:hover{text-transform:uppercase;color: #9090F0;}
        </style>
</head>
```
Though this HTML code can be considered valid (a standalone browser can render it), the built-in browser doesn't like it:
![XmlNotepad-META-error](https://user-images.githubusercontent.com/20116984/127382210-30d30383-d5dc-4fd9-a6c2-c30fb8ca8832.png)

OS: Windows 10 x64 (Version	10.0.18362)
App: 2.8.0.40

**Comment by lovettchris** [comment]
Can you share your XSLT stylesheet?  I wonder if adding <xsl:output method="html"/> solves the problem?

**Comment by Anton-V-K** [comment]
Thanks for loooking into this.
The XSL-file already contais this statement:
```
    <xsl:output method="html"  encoding="utf-16"/>
```
Here is the archive with XML and XSL - [ActivityLog.zip](https://github.com/microsoft/XmlNotepad/files/6911131/ActivityLog.zip)

**Comment by lovettchris** [comment]
This will be fixed in 2.8.0.41.
